packages/training/deploy/runpod/setup.py

```
@@ -188,20 +230,98 @@ def cmd_logs(args):
+    print(f"\n✓ Created benchmark pod: {pod['id']}")
+    print(f"  Name: {args.name or f'babylon-bench-{args.gpu}'}")
+    print(f"  GPU: {gpu_id}")
+    if args.hf_model:
+        print(f"  HF Model: {args.hf_model}")
+    if args.model:
+        print(f"  Model Path: {args.model}")
+    print(f"  Quick mode: {args.quick}")
+    if args.scenario:
+        print(f"  Scenario: {args.scenario}")
+    print(f"  Spot: {args.spot}")
+    print(f"\n  View at: https://console.runpod.io/pods")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)